modules/apps/frontend-theme/frontend-theme-dialect/src/css/dialect/variables/_navs.scss

```diff
@@ -0,0 +1,387 @@
+// nav-base-link
+
+$nav-base-link: (
+	after: (
+		bottom: 0,
+		content: none,
+		display: block,
+		position: absolute,
+		left: 0,
+		right: 0,
+		width: auto,
+	),
+);
+
+// Primary tab
```

Comment:
Also it might be better to call this `Nav Primary` since these classes can be applied to navs not just tabs and this is the language Clay is usually using.

Reply:
If you decide to call it Nav instead of Tab make sure to apply that consistently to the other headings.

Reply:
Fixed